packages/venia-ui/lib/components/Header/header.css

```diff
@@ -46,10 +55,17 @@
 
 .secondaryActions {
     grid-area: secondary;
-    width: 100%;
+    width: max-content;
     display: grid;
-    grid-template-columns: 1fr 1fr;
-    justify-items: end;
+    grid-auto-flow: column;
+    column-width: max-content;
```

Comment:
I don't believe this is a grid property.
```suggestion
```

Reply:
True. Fixed